cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/AnomalyDetectorConfig.java

```diff
@@ -172,6 +173,22 @@ private AnomalyDetectorConfig() {
   public static final String ANOMALY_DETECTION_ALLOW_CAPACITY_ESTIMATION_DOC = "The flag to indicate whether anomaly "
       + "detection threads allow capacity estimation in the generated cluster model they use.";
 
+  /**
+   * <code>topic.anomaly.detection.interval.ms</code>
+   */
+  public static final String TOPIC_ANOMALY_DETECTION_INTERVAL_MS_CONFIG = "topic.anomaly.detection.interval.ms";
+  public static final String TOPIC_ANOMALY_DETECTION_INTERVAL_MS_DOC = "The interval in millisecond that topic anomaly "
+      + "detector will run to detect topic anomalies. If this interval time is not specified, topic anomaly detector "
+      + "will run with interval specified in " + ANOMALY_DETECTION_INTERVAL_MS_CONFIG + ".";
+
+  /**
+   * <code>topic.anomaly.finder.class</code>
+   */
+  public static final String TOPIC_ANOMALY_FINDER_CLASSES_CONFIG = "topic.anomaly.finder.class";
+  public static final String DEFAULT_TOPIC_ANOMALY_FINDER_CLASS = TopicReplicationFactorAnomalyFinder.class.getName();
```

Comment:
If we make this the default topic anomaly finder, even if the user does not want to use the topic anomaly finder, s/he will receive an `IllegalXXX` exception upon configuration time due to missing default replication factor. Should we:
1. Make the NoopTopic Anomaly Finder as the default here, and 
2. Add a documentation under the wiki on what needs to be done to use `TopicReplicationFactorAnomalyFinder`?
3. Also add documentation on configurations section of wiki to add the new configs?